pgrx/src/datum/geo.rs

```diff
@@ -21,17 +42,17 @@ impl FromDatum for pg_sys::BOX {
         if is_null {
             None
         } else {
-            let the_box = datum.cast_mut_ptr::<pg_sys::BOX>();
+            let the_box = datum.cast_mut_ptr::<Self>();
             Some(the_box.read())
         }
     }
 }
 
-impl IntoDatum for pg_sys::BOX {
+impl IntoDatum for Box {
     fn into_datum(mut self) -> Option<pg_sys::Datum> {
         unsafe {
             let ptr = PgMemoryContexts::CurrentMemoryContext
-                .copy_ptr_into(&mut self, std::mem::size_of::<pg_sys::BOX>());
+                .copy_ptr_into(&mut self, std::mem::size_of::<Self>());
```

Comment:
While we're here, let's enforce this style:
```suggestion
                .copy_ptr_into(&mut self, mem::size_of::<Self>());
```